addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -110,9 +110,16 @@
         <div t-attf-class="text-center d-flex align-items-center text-center pb-1 #{'justify-content-between' if product_info['has_discounted_price'] else 'justify-content-around'}">
             <div class="css_editable_mode_hidden">
                 <!-- real price -->
-                <div class="oe_price fw-bold text-nowrap h2 my-2"
-                     t-esc="product_info['price']"
-                     t-options="{'widget': 'monetary', 'display_currency': product_info['currency']}"/>
+                <t t-if="len(str(product_info['price']))>10">
+                    <div class="oe_price fw-bold text-nowrap h4 my-2"
+                        t-esc="product_info['price']"
+                        t-options="{'widget': 'monetary', 'display_currency': product_info['currency']}"/>
+                </t>
+                <t t-else="">
+                    <div class="oe_price fw-bold text-nowrap h2 my-2"
```

Comment:
in my opinion it would be better to just have a `t-attf-class` or `t-att-class` (or a `style="zoom:0.8"` or something like that) than duplicate the element (duplicating the element makes it more complicated to do a customization, you now have to do 2 XPATH to apply it for both case)

Reply:
yeah, that's so much better, thanks for the tip!